.editorconfig

```diff
@@ -11,7 +11,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
do you have latest master merged into your branch? not sure why this is showing as a diff, its already `4` on master.

Reply:
I probably don't, but its still letting you merge because there are no conflicts